tests/operator/remember.ts

```diff
@@ -131,4 +131,24 @@ describe('Stream.prototype.remember', () => {
     });
     done();
   });
+
+  it ('should work properly with `xs.create()`', (done) => {
+    const stream = xs.create().remember();
```

Comment:
This should be changed to create a reference to the `xs.create` stream and another references to the external `remember` stream.  The `shamefullySendNext` should be on the internal stream.  So it should read `const stream = xs.create(); const external = stream.remember();`